.env

```diff
@@ -5,4 +5,4 @@ VITE_APP_ZITADEL_SILENT_REDIRECT_URI='/authentication/silent-callback'
 VITE_APP_ZITADEL_AUTHORITY_URL='http://localhost:8081'
 VITE_APP_ZITADEL_CLIENT_ID=''
 VITE_APP_AUTH_USE_SERVICE_WORKER_ONLY=false
-VITE_APP_AUTH_DISABLED=true
+VITE_APP_AUTH_DISABLED=false
```

Comment:
@baixinsui the idea was to have .env with local oauth and .env.zitadel-testbed with zitadel testbed oauth.
Otherwise, we cannot provide a local zitadel configuration.

 VITE_APP_AUTH_DISABLED=true can be put in any .env file.

Reply:
@baixinsui can you please check if the change what i have added now fits to your expectation?

Reply:
LGTM